src/System.Numerics.Tensors/Directory.Build.props

```diff
@@ -4,5 +4,7 @@
     <PackageVersion>0.3.0</PackageVersion>
     <AssemblyVersion>0.2.0.0</AssemblyVersion>
     <StrongNameKeyId>Open</StrongNameKeyId>
+    <!-- This is a preview package. Do not mark as stable. -->
+    <BlockStable>true</BlockStable>
```

Comment:
We should also update the packageIndex.json:
https://github.com/dotnet/corefx/blob/dd3ad315e64a40f5954e96793cdecd61d1173aa7/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L3172-L3182

Reply:
Also, fix harvestPackages.props:
https://github.com/dotnet/corefx/blob/40f031a7c7a8241f92a9f6d7f269dc99228d2893/external/harvestPackages/harvestPackages.props#L109-L111

Reply:
No we shouldn't do either of these things.  Why would we do that?

Reply:
Oh I misunderstood. I thought the packageIndex does not already those versions. (reacting to my reaction)